src/v2/Apps/Artwork/Components/ArtworkRelatedArtists.tsx

```diff
@@ -53,9 +53,9 @@ export const ArtworkRelatedArtists: React.FC<ArtworkRelatedArtistsProps> = track
     return (
       <Box mt={6} data-test={ContextModule.relatedArtistsRail}>
         <Flex flexDirection="column" alignItems="center">
-          <Serif size={["5t", "8"]} color="black100" mb={2} textAlign="center">
```

Comment:
I did this as well (dropped the responsive props) on the assumption that `Text` will do the desired responsive thing. 

I'm assuming that's fine, but should have checked beforehand!